libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -59,8 +59,15 @@ class AP_ExternalAHRS {
     // Get model/type name
     const char* get_name() const;
 
+    enum class AvailableSensor {
```

Comment:
you can gain some bytes by typing the enum to uint16_t
enum class AvailableSensor : uint16_t